stubs/console-menu/consolemenu/prompt_utils.pyi

```diff
@@ -0,0 +1,46 @@
+from typing import Any, NamedTuple
+
+from consolemenu.screen import Screen
+from consolemenu.validators.base import BaseValidator as BaseValidator, InvalidValidator as InvalidValidator
```

Comment:
Are these really intended to be re-exported?

Reply:
I don't really understand re-exporting in Python, could you please explain what you mean? From what I've read in the Contributing Guidelines, packages intended to be re-exported are denoted using the `as` statement, usually using the same identifier as the import statement. Beyond that I'm a little confused.

Reply:
The re-exports use the correct syntax, but they seem to be unnecessary. The purpose of a re-export is to make it so that you can import a name from the module that defines it or from the module that re-exports it. But you probably aren't supposed to do `from consolemenu.prompt_utils import BaseValidator`, you should import from `consolemenu.validators.base` instead. So these re-exports are unnecessary and should be removed.

Reply:
The latest commit should, if I am understanding correctly, have this resolved.